constants/index.ts

```diff
@@ -166,6 +166,8 @@ export const NETWORK_TICKERS: KeyValueT<string> = {
   bitcoincash: 'BCH'
 }
 
+export type NetworkTickersType = 'XEC' | 'BCH'
```

Comment:
I'm pretty wary of stuff like this but we can leave it if Estevao thinks it's fine.

I made this phase 3 task though: https://github.com/PayButton/paybutton-server/issues/859